cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js

```diff
@@ -68,13 +68,27 @@ export default function AddOpportunity() {
   const [changePage, setChangePage] = useState(false);
   const [localityName, setLocalityName] = useState("Your Community");
 
-  // Handle Modal if proximity data is still loading
+  // Handle Modal for errors
   const [show, setShow] = useState(false);
   const handleClose = () => {
     setShow(false);
   };
   const handleShow = () => setShow(true);
 
+  // handle jproximity data still loading modal
+  const [proxModalShow, setProxModalShow] = useState(false);
```

Comment:
Is "jproximity" a typo?
```suggestion
  // handle proximity data still loading modal
  const [proxModalShow, setProxModalShow] = useState(false);
```